packages/mediacenter/kodi/package.mk

```diff
@@ -406,6 +405,22 @@ post_makeinstall_target() {
       cp $PKG_DIR/config/appliance.xml $INSTALL/usr/share/kodi/system/settings
     fi
 
+  # update addon manifest
+  xmlstarlet ed -L -d "/addons/addon[text()='service.xbmc.versioncheck']" \
+    $INSTALL/usr/share/kodi/system/addon-manifest.xml || :
```

Comment:
Can you remove `|| :` from the `xmlstarlet` calls as it shouldn't fail silently - thanks.